.bazelrc

```diff
@@ -0,0 +1 @@
+build --//:mypy=//third_party:mypy
```

Comment:
if we use a valid default for the mypy version then we don't have to set this here